app/graphql/mutations/eras/update_era.rb

```diff
@@ -3,12 +3,16 @@ class Mutations::Eras::UpdateEra < ::Mutations::BaseMutation
   argument :name, String, required: false
   argument :start_date, String, required: false
   argument :end_date, String, required: false
+  argument :week_number, String, required: false
```

Comment:
Same here that I don't think we'd accept week_num as an arg?

Reply:
Agreed.

Reply:
It's outta there!